### DISCUSSION TITLE
Parse the known_*.log and software.log into SO

### DISCUSSION THREAD

**Discussion by 0nnyx** [Security-Onion-Solutions/securityonion#10265] [Unsupported Versions]
So far, the known_*.log and software.log files (https://docs.zeek.org/en/master/logs/known-and-software.html) are generated by default by Zeek but not parsed by Kibana / SO while this info is useful. Once these are parsed, please also add default OQL queries to view them in the Hunt.

**Comment by dougburks** [comment]
Zeek software.log should work in the new Security Onion 2.4:
https://github.com/Security-Onion-Solutions/securityonion/blob/2.4/main/salt/elasticsearch/files/ingest/zeek.software

known_*.log would require additional manual configuration.
